src/components/dynamic/Radio.jsx

```diff
@@ -1,13 +1,24 @@
 import { BsCircleFill, BsCircle } from "react-icons/bs";
 
-const Radio = ({ text, field, options, user, setUser, editable = true }) => {
+const Radio = ({
+  text,
+  field,
+  options,
+  user,
+  setUser,
+  editable = true,
+  required,
+}) => {
   const handleClick = (option) => {
     setUser({ ...user, [field]: option });
   };
 
   return (
     <div className="flex flex-col">
-      <p className="mb-1 font-semibold">{text}</p>
+      <p className="mb-1 font-semibold">
+        {text}{" "}
```

Comment:
remove the space?!

Reply:
ahhh I think npm run format added that by itself. lemme remove